packages/mobile/src/harmony-native/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -72,29 +98,78 @@ export const BaseButton = (props: BaseButtonProps) => {
   }
 
   const animatedStyles = useAnimatedStyle(() => ({
-    transform: [{ scale: interpolate(pressed.value, [0, 1], [1, 0.97]) }]
+    transform: [{ scale: interpolate(pressed.value, [0, 1], [1, 0.97]) }],
+    ...(dynamicStyles?.press.border
```

Comment:
Did you see what was done for iconbutton, where the animations specific to icon button were defined in that component, and then those styles plus the animatedRef was passed through? That way you can avoid dynamic styles and just pass styles prop in like we do in desktop. It might be more clear for the caller to define the animations basically? Just a thought cause this way also works